packages/near-membrane-base/src/red.ts

```diff
@@ -183,11 +178,16 @@ export const serializedRedEnvSourceText = (function redEnvFactory(blueEnv: Membr
     }
 
     function getDistortedValue(target: RedProxyTarget): RedProxyTarget {
-        if (!WeakMapHas(distortionMap, target)) {
-            return target;
+        let distortedTarget: RedProxyTarget = target;
+        try {
```

Comment:
we need the try/catch here because if the callback throws, the error is from the blue realm, that is not good... also, what does it mean to throw when trying to distort something?

Reply:
> what does it mean to throw when trying to distort something?

Likely a critical fail we should know about, so a good place to instrument.